engine/playground-render-canvas/package.json

```
@@ -0,0 +1,40 @@
+{
+	"name": "playground-render-canvas",
+	"version": "0.0.1",
+	"description": "Extension for the Slang Shading Language",
+	"main": "dist/index.umd.js",
+	"module": "dist/index.es.js",
+	"types": "dist/index.d.ts",
+	"exports": {
+		".": {
+			"types": "./dist/index.d.ts",
+			"style": "./dist/playground-render-canvas.css",
+			"import": "./dist/index.es.js",
+			"require": "./dist/index.umd.js"
+		},
+		"./dist/playground-render-canvas.css": {
+			"default": "./dist/playground-render-canvas.css"
+		}
+	},
+	"scripts": {
+		"build": "vite build",
+		"type-check": "vue-tsc --build"
+	},
+	"dependencies": {
+		"slang-playground-shared": "file:../shared"
+	},
```

If you don't plan to publish this to npm then this should work fine I think. But if you do, this is not the right way to do things, since it wouldn't work; you'd want to instead treat it as just another regular dependency, like this:

```suggestion
	"dependencies": {
		"slang-playground-shared": "0.0.1"
	},
```

Same applies for other `file:` dependencies in this repo. But again, if you're not publishing to npm then I don't think this is an actual problem.